src/Sulu/Bundle/DocumentManagerBundle/Command/FixturesLoadCommand.php

```diff
@@ -113,7 +113,13 @@ function(BundleInterface $bundle) {
                 $this->kernel->getBundles()
             );
 
-            $directories[] = $this->kernel->getRootDir();
+            if (method_exists($this->kernel, 'getRootDir')) {
+                $directories[] = $this->kernel->getRootDir();
+            }
+
+            $directories[] = $this->kernel->getProjectDir();
+            $directories[] = $this->kernel->getProjectDir() . '/src';
```

Comment:
I find this detection based on the directory should be deprecated. We should go for service injection like its also done for doctrine fixture bundle since [3.0](https://github.com/doctrine/DoctrineFixturesBundle/blob/master/UPGRADE.md#upgrade-to-30)

@sulu/core-team what do you think defining a `sulu_document_manager.fixture` tag and autowire the `DocumentFixtureInterface` to it so we can inject it here.

Reply:
Sounds good to me. Would just call the tag `sulu.document_manager_fixture`, because prefixing everything with `sulu.` is more consistent IMO.